bindings/core/src/method_handler/account.rs

```diff
@@ -309,10 +309,7 @@ pub(crate) async fn call_account_method_internal(account: &Account, method: Acco
         } => {
             let transaction = account
                 .sign_and_submit_transaction(
-                    PreparedTransactionData::try_from_dto(
-                        prepared_transaction_data,
-                        &account.client().get_protocol_parameters().await?,
-                    )?,
+                    PreparedTransactionData::try_from_dto_unverified(prepared_transaction_data)?,
```

Comment:
Because we don't have protocol parameters offline?

Reply:
Yeah we only have the default values which can be wrong

Reply:
But we're losing the verif also with online clients, is that an issue ?